Mathlib/RingTheory/MvPowerSeries/Inverse.lean

```diff
@@ -137,6 +138,21 @@ theorem mul_invOfUnit (φ : MvPowerSeries σ R) (u : Rˣ) (h : constantCoeff σ
         exact Nat.eq_zero_of_le_zero (H s)
 #align mv_power_series.mul_inv_of_unit MvPowerSeries.mul_invOfUnit
 
+@[simp]
+theorem invOfUnit_mul (φ : MvPowerSeries σ R) (u : Rˣ) (h : constantCoeff σ R φ = u) :
```

Comment:
Here `φ` and `u` can maybe be implicit, but do as you think it is more convenient.

Reply:
The point is that φ and u are not implicit in the preexisting `mul_invOfUnit`.

Reply:
OK, let's keep it as it is then.

Reply:
Also, we discussed with Filippo that it could be useful to have it explicit, in case where one wants lean to produce the hypothesis `h`. 
(TODO?: prove equivalence)